docs/source/environment.rst

```diff
@@ -0,0 +1,150 @@
+.. _Environment:
+
+Environment
+***********
+This page describes the environments in which Fab can be run.
+
+
+.. _Requirements:
+
+Requirements
+============
+The minimum dependencies for Fab to build a project are:
+
+* Python >= 3.7
+* Compiler
+* Linker (or archiver, if building libraries)
+
+If your project includes C code:
+
+* libclang
+* python-clang
+
+
+Running Fab at The Met Office
+=============================
+
+VDI
+---
+
+Without clang
+^^^^^^^^^^^^^
+If you don't need to build C, you can use this modules command::
+
+    $ module load fab/0.11.0/python/3.7.0
```

Comment:
It may be necessary to use the Extra Software collection: `module use $EXTRA_SOFTWARE`. This line wont work unless you define the variable but it is indicative.